src/Aether/Devices/Sensors/ObservableSgp4x.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Aether.Devices.Sensors.Metadata;
+using System;
+using System.Collections.Generic;
+using System.Device.I2c;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnitsNet;
+
+namespace Aether.Devices.Sensors
+{
+    internal class ObservableSgp4x : ObservableSensor, IObservableI2cSensorFactory
+    {
+        private readonly Drivers.Sgp4x _sensor;
+
+        private ObservableSgp4x(I2cDevice device)
+        {
+            _sensor = new Drivers.Sgp4x(device);
+            Start();
+        }
+
+        public static int DefaultAddress => Drivers.Sgp4x.DefaultI2cAddress;
+
+        public static string Manufacturer => "Sensirion";
+
+        public static string Name => "SGP4x";
+
+        public static string Uri => "https://www.sensirion.com/en/environmental-sensors/gas-sensors/sgp40/";
+
+        public static IEnumerable<MeasureInfo> Measures { get; } = new[]
+        {
+            new MeasureInfo(Measure.VOC)
+        };
+
+        public static IEnumerable<SensorDependency> Dependencies => SensorDependency.NoDependencies;
```

Comment:
This is where you'd add a humidity & temperature dependency.

The SCD4x has a dependency on barometric pressure if you want to see how this and the `ProcessLoopAsync` changes for that.

Reply:
Done.